packages/react-components/react-dialog/src/components/Dialog/renderDialog.tsx

```diff
@@ -1,10 +1,11 @@
 /** @jsxRuntime automatic */
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31132&commitId=061009b0f54e43657ac47212546bb9321cd4adc8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.image-bad-url+icon.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Dialog</b> 47 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.actions position start & position end.chromium.png|1267| Changed|
|Dialog.actions position start - High Contrast.chromium.png|3622| Changed|
|Dialog.alert - RTL.chromium.png|1267| Changed|
|Dialog.actions position start & position end - Dark Mode.chromium.png|3622| Changed|
|Dialog.alert - High Contrast.chromium.png|3622| Changed|
|Dialog.default - Dark Mode.chromium.png|3622| Changed|
|Dialog.actions position start - RTL.chromium.png|1267| Changed|
|Dialog.alert - Dark Mode.chromium.png|3622| Changed|
|Dialog.actions position start - Dark Mode.chromium.png|3622| Changed|
|Dialog.alert.chromium.png|1267| Changed|
|Dialog.actions position start & position end - RTL.chromium.png|1267| Changed|
|Dialog.Integration Combobox Inline.chromium.png|738| Changed|
|Dialog.Fluid Actions Start.chromium.png|1267| Changed|
|Dialog.actions position start.chromium.png|1267| Changed|
|Dialog.actions position start & position end - High Contrast.chromium.png|3622| Changed|
|Dialog.Fluid Actions End.chromium.png|1267| Changed|
|Dialog.default - High Contrast.chromium.png|3622| Changed|
|Dialog.default.chromium.png|1267| Changed|
|Dialog.no actions - Dark Mode.chromium.png|3494| Changed|
|Dialog.nested - RTL.chromium.png|2732| Changed|
|Dialog.nested.chromium.png|2732| Changed|
|Dialog.no title & no actions - RTL.chromium.png|1220| Changed|
|Dialog.no actions - RTL.chromium.png|1240| Changed|
|Dialog.no actions - High Contrast.chromium.png|3494| Changed|
|Dialog.nested - Dark Mode.chromium.png|6066| Changed|
|Dialog.default - RTL.chromium.png|1267| Changed|
|Dialog.nested - High Contrast.chromium.png|6067| Changed|
|Dialog.no actions.chromium.png|1240| Changed|
|Dialog.no title & no actions.chromium.png|1220| Changed|
|Dialog.no title - High Contrast.chromium.png|3533| Changed|
|Dialog.no title & no actions - High Contrast.chromium.png|3405| Changed|
|Dialog.no title & no actions - Dark Mode.chromium.png|3405| Changed|
|Dialog.no title - Dark Mode.chromium.png|3532| Changed|
|Dialog.no title.chromium.png|1248| Changed|
|Dialog.no title - RTL.chromium.png|1248| Changed|
|Dialog.non-modal.chromium.png|2700| Changed|
|Dialog.non-modal - Dark Mode.chromium.png|6510| Changed|
|Dialog.scroll long content - RTL.chromium.png|611| Changed|
|Dialog.non-modal - RTL.chromium.png|2700| Changed|
|Dialog.non-modal - High Contrast.chromium.png|6511| Changed|
|Dialog.scroll long content - High Contrast.chromium.png|2895| Changed|
|Dialog.scroll long content - Dark Mode.chromium.png|2894| Changed|
|Dialog.title custom action - RTL.chromium.png|1242| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|3507| Changed|
|Dialog.scroll long content.chromium.png|611| Changed|
|Dialog.title custom action.chromium.png|1242| Changed|
|Dialog.title custom action - High Contrast.chromium.png|3507| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>